test/e2e/flowaggregator_test.go

```diff
@@ -994,3 +1014,22 @@ func deletePerftestServices(t *testing.T, data *TestData) {
 		}
 	}
 }
+
+// getBandwidthAndSourcePort parses iperf commands output and returns bandwidth
+// and source port.
```

Comment:
maybe add that the bandwidth is returned as a slice containing 2 strings (bandwidth value and bandwidth unit)